torch/_torch_docs.py

```diff
@@ -5646,7 +5646,9 @@ def merge_dicts(*dicts):
     device (:class:`torch.device`, optional): the desired device of returned tensor.
         Default: if None, uses the current device for the default tensor type
```

Comment:
This documentation is outdated, since we don't have a notion of "current device for the default tensor type" anymore. Plus, with your change, when device is None then PyTorch will produce a sparse tensor on the same device as the values tensor.

Reply:
(The above note is outdated based on offline discussion.)

Reply:
Thanks, Mike! I updated the PR based on latest feedback. cc @mruberry